packages/ComponentLibrary/src/components/Table/sections/LeftSection.tsx

```diff
@@ -0,0 +1,17 @@
+import IconButton from '../../IconButton';
+import ChevronDown from '../../../assets/icons/chevrons-down.svg';
+import Refresh from '../../../assets/icons/refresh-cw.svg';
+import { Box } from '@mui/material';
+
+const LeftSection = () => (
+  <Box sx={{ display: 'flex', gap: '0.25rem' }}>
+    <IconButton tooltip="New Line" onClick={() => {}} width={16} height={16}>
```

Comment:
Move styles to another file & create an array to map them and not repeat so much of the same code